tests/integration/mocks/contacts.js

```diff
@@ -3,16 +3,32 @@ var mock = {}
 mock = {
     contact_id: "place",
     contact : {
-        name: "dave",
-        vat_number: "456ghtd",
-        city: "london",
-        telephone: "0203467893"
+        mainObject: {
```

Comment:
What was the rational for naming things from `order`, `units, `contacts`, etc to `mainObject` and `minorObject`.